covid_data_vac.py

```diff
@@ -800,3 +800,28 @@ def vac_slides():
             df = vac_manuf_given(df, page, file, i, link)
             #df = vac_slides_groups(df, page, file, i)
     return df
+
+def mobility():
```

Comment:
I think it makes sense to make this about IHME data and give us the all the data for thailand, or most of it. ihme_dataset()?
Also move this to covid_data_api since that main deals with simple data thats in csv or json.

Reply:
There are a lot of columns in IHME data that overlaps what's already in the project, and the numbers probably do not line up. (Data dictionary: https://ihmecovid19storage.blob.core.windows.net/latest/data_dictionary.csv)

Any preference on whether to include or exclude those? (e.g. cases, deaths, tests)

Reply:
I'll just keep them all for now, we can pick and choose later.